sympy/sets/ordinals.py

```diff
@@ -1,36 +1,67 @@
 from sympy.core import Basic
 import operator
 
+class OrdinalPow(Basic):
+    """
+    Represents ordinal exponential and multiplication terms one of the
+    building blocks of the Ordinal class.
+    In OrdinalPow(a, b) a represents exponent and b represents multiplicity.
+    """
+    def __new__(cls, *terms):
+        if not all (isinstance(i, (int, Ordinal)) for i in terms):
```

Comment:
Perhaps class `Integer` could also be accepted.

Reply:
Is it declining `Integer` now ?

Reply:
It is not included in the list.

Reply:
could you please provide an example?

Reply:
```
>>> isinstance(Integer(2), (int, Ordinal))
False
```